server/service/src/processors/transfer/requisition/create_response_requisition.rs

```diff
@@ -359,3 +396,91 @@ fn generate_response_requisition_indicator_values(
     }
     Ok(vec![])
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use chrono::{NaiveDate, NaiveDateTime};
+    use repository::{
+        mock::{
+            mock_name_b, mock_request_draft_requisition, mock_store_b, MockData, MockDataInserts,
+        },
+        test_db::setup_all_with_data,
+    };
+
+    #[actix_rt::test]
+    async fn test_create_inbound_requisition_picked_cutoff() {
+        let requisition_row_old = RequisitionRow {
+            id: "requisition_row_old".to_string(),
+            status: RequisitionStatus::Sent,
+            created_datetime: NaiveDateTime::new(
+                NaiveDate::from_ymd_opt(2020, 6, 6).unwrap(),
+                Default::default(),
+            ),
+            sent_datetime: Some(NaiveDateTime::new(
+                NaiveDate::from_ymd_opt(2020, 6, 6).unwrap(),
+                Default::default(),
+            )),
+            ..mock_request_draft_requisition()
+        };
+        let requisition_old = Requisition {
+            requisition_row: requisition_row_old.clone(),
+            name_row: mock_name_b(),
+            store_row: mock_store_b(),
+            program: None,
+            period: None,
+        };
+        let requisition_transfer_old = RequisitionTransferProcessorRecord {
+            other_party_store_id: "store_a".to_string(),
+            requisition: requisition_old.clone(),
+            linked_requisition: None,
+        };
+
+        let requisition_row_new = RequisitionRow {
+            id: "requisition_row_new".to_string(),
+            sent_datetime: Some(
+                NaiveDate::from_ymd_opt(2025, 8, 7)
```

Comment:
will this test fail in a few months time?

Reply:
Oh I see, it won't, bc earliest sync log was 2024...

Wondering if better to insert a mock sync log here, than the base mock data you added - so we can see how it lines up with the dates of your test invoices? Kind of hard to see how it relates 👀 

Extra excuse is if someone adds a super old sync log to the shared mock data at some point, this test would fail 😱

Reply:
>Wondering if better to insert a mock sync log here, than the base mock data you added - so we can see how it lines up with the dates of your test invoices/requisitions? Kind of hard to see how "new" and "old" are defined 👀

Yea I generally hate the base mock data for this reason 😆 

>Extra excuse is if someone adds a super old sync log to the shared mock data at some point, this test would fail 😱

That's a THEM problem

Jokes I'll just copy pasta the logs into this test so they're safe

Reply:
I feel like base mock data is something that would be 'shared' amongst test, do you reckon these two mock records would be useful in future tests somewhere ?

Reply:
Not sure, maybe if someone needs something quick and dirty? Can just nuke them.